src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -126,28 +123,25 @@ const CpsRecommendations = ({ items, parentColumns }) => {
 
   return (
     <GridItemConstrainedMediumNoMargin>
-      <SkipLinkWrapper
-        service={service}
-        endTextId={endTextId}
-        text={text}
-        endTextVisuallyHidden={endTextVisuallyHidden}
-        terms={skipLinkTerms}
-      >
-        <RecommendationsWrapper>
-          <CpsOnwardJourney
-            labelId="recommendations-heading"
-            title={title}
-            content={items}
-            parentColumns={parentColumns}
-            singleTransform={singleTransform}
-            listTransform={listTransform}
-            sectionLabelOverrideAs="strong"
-            sectionLabelBar={false}
-            sectionLabelBackground={C_LUNAR}
-            columnType="main"
-          />
-        </RecommendationsWrapper>
-      </SkipLinkWrapper>
+      <RecommendationsWrapper>
+        <CpsOnwardJourney
+          labelId="recommendations-heading"
+          title={title}
+          content={items}
+          parentColumns={parentColumns}
+          singleTransform={singleTransform}
+          listTransform={listTransform}
+          sectionLabelOverrideAs="strong"
+          sectionLabelBar={false}
+          sectionLabelBackground={C_LUNAR}
+          columnType="main"
+          hasSkipLnk
+          endTextId={endTextId}
```

Comment:
Some of the naming of these props now that they're directly being passed into `CpsOnwardJourney` aren't very clear they're being used for the skip link. Might need to think about renaming them to be more explicit.

Reply:
this is now grouped them in a `skipLink` object so the relationship would be clearer